### DISCUSSION TITLE
Number of sessions in the statistics

### DISCUSSION THREAD

**Discussion by radio5050** [AzuraCast/AzuraCast#6632] [Ideas]
Hello and really thank you for your work.

Do you know what would be perfect for the statistics? It would be the number of sessions or connexion.
It is a statistic which is used in France and it would permit to compare with others french radio.
To be clear it is the number of connexion, if a unique auditeur connect 10 times a day, it will count 10 sessions

Thanks gain

**Comment by BusterNeece** [comment]
If you don't mind, please consider submitting this over at https://features.azuracast.com/ as a feature request.
